lib/mobility-core/src/Kernel/External/AadhaarVerification/Gridline/Types.hs

```diff
@@ -67,9 +67,16 @@ data GridlineVerifyAadhaarResp = GridlineVerifyAadhaarResp
 data GridlineVerifyAadhaarData = GridlineVerifyAadhaarData
   { code :: Text,
     message :: Text,
-    transaction_id :: Text
+    transaction_id :: Maybe Text
   }
-  deriving (Show, Generic, ToJSON, FromJSON)
+  deriving (Show, Generic, ToJSON)
+
+instance FromJSON GridlineVerifyAadhaarData where
+  parseJSON = withObject "GridlineVerifyAadhaarData" $ \v -> do
```

Comment:
Why do you need to write FromJSON like this?

Reply:
because transaction id will be a maybe type field because there api is giving status as 200 but  giving error so when it will have in 200 they are not giving transaction id  so   i made it as maybe type field so to parse maybe type value i have done like this.